Rakefile

```diff
@@ -36,4 +36,4 @@ Rake::TestTask.new do |t|
   t.verbose = true
 end
 
-task :default => :test
+task default: [:test, :rubocop]
```

Comment:
@bf4 I think a task `ci` do not solves the problem, since the default command when we clone a project and want to check if everything is ok is `$ rake`. And as @NullVoxPopuli said we still can run only test if we want.

What your concern about it? The status code is preserved so, if the tests fail the rubocop does not run and the CI will fail.